lib/text.js

```diff
@@ -9,13 +9,16 @@ const wrapBold = ({bold}, el) => bold ? <b>{el}</b> : el;
 
 const wrapItalic = ({italic}, el) => italic ? <i>{el}</i> : el;
 
+const wrapStrikethrough = ({strikethrough}, el) => strikethrough ? <s>{el}</s> : el;
```

Comment:
wrapping ternary in parens should be good for readability (I think airbnb style requires it)